actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorProfiler.java

```diff
@@ -44,7 +43,7 @@
 {
     private final HashMap<Object, ProfilerData> collectedData = new HashMap<>();
     private final ReadWriteLock rwlock = new ReentrantReadWriteLock();
-    private final KryoCloner kryoCloner = new KryoCloner();
+    private final KryoSerializer kryoCloner = new KryoSerializer();
```

Comment:
This class actually just uses this to clone objects for it's own profiling, so it's not actually a generic field for any serializer/cloner, it's specifically for this profiler.